py3status/modules/xrandr.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Control your screen(s) layout easily.
+Control your screen layout easily.
```

Comment:
I looked it up. It seems like people don't use `screens layout`. It's either `Control your screens easily` or `Control your screen layout easily.` I avoid (s) to look clean.